core/templates/domain/story/editable-story-backend-api.service.ts

```diff
@@ -52,125 +65,123 @@ angular.module('oppia').factory('EditableStoryBackendApiService', [
             classroomUrlFragment: classroomUrlFragment
           });
         }
-      }, function(errorResponse) {
+      }, (errorResponse) => {
         if (errorCallback) {
-          errorCallback(errorResponse.data);
+          errorCallback(
+            errorResponse.error
+          );
         }
       });
-    };
-
-    var _updateStory = function(
-        storyId, storyVersion, commitMessage, changeList,
-        successCallback, errorCallback) {
-      var editableStoryDataUrl = UrlInterpolationService.interpolateUrl(
-        EDITABLE_STORY_DATA_URL_TEMPLATE, {
+  }
+    private _updateStory = function(
+        storyId: string, storyVersion: string,
+        commitMessage: string,
+        changeList: string[],
```

Comment:
Is `string[]` the correct type for `changeList`?

Reply:
@nishantwrp yes maybe

Reply:
Nope, changeList is not an array of `string`. It's type maybe is
https://github.com/oppia/oppia/blob/e8efb007d2bbd5aea913893fc9806dbb5f5368e3/core/templates/domain/editor/undo_redo/ChangeObjectFactory.ts#L402